### ISSUE TITLE
Missing Icon

### ISSUE THREAD

**Issue by fmarier** [allan-simon/verbiste-android#5]
The app uses the default Android icon both in F-Droid and in the list of apps once it's installed.

It would be easier to recognize if it used a distinctive icon :)

I would suggest simply reusing the Verbiste icon: https://sources.debian.net/data/main/v/verbiste/0.1.44-1/src/gtk/verbiste-512x512.png

**Comment by allan-simon** [comment]
You are absolutely right, and actually I don't know how i've missed that verbiste already had it's own icon
